lib/IDE/CodeCompletion.cpp

```diff
@@ -3251,6 +3333,10 @@ class CompletionLookup final : public swift::VisibleDeclConsumer {
       // Optional<T> type.
       resultTy = OptionalType::get(resultTy);
     }
+
+    if (implictlyAsync)
+      Builder.addAnnotatedAsync();
```

Comment:
Is it possible to insert an `await` when we have a call to async functions, whether implicitly (like here) or one that is defined as `async`? Like when we have code completion on an optional value, e.g.,

```swift
struct X {
  var member: Int
}
var xOpt: X?
xOpt.<#completion here#>  // produces a completion with "member" that also adds a ? before the .
```

Reply:
That would be great, we also want to do it for `try` as well.
It should be technically feasible. But unlike `?` for optionals, we need to insert them to arbitrary point. For example, for
```
_ = 1 + foo.bar {
  /* ... */
} .<HERE>
```
We need to insert `try`/`await` at the line not the cursor is on.
Also, we need to decide where to insert them. i.e. `await _ = 1 + foo.bar ...` or `_ = await 1 + foo.bar ...`